utils/uiTypes/VotePlugin.ts

```diff
@@ -148,21 +151,26 @@ export class VotingClient {
       this.clientType = VotingClientType.GatewayClient
       this.noClient = false
     }
+    if (this.client instanceof PythClient) {
```

Comment:
instead of adding onto a monolithic class, how about making a new class that implements a common interface

Reply:
I really don't understand what this means, do you care to explain or give an example of someone that integrated this way?